argilla/docs/tutorials/token_classification.ipynb

```diff
@@ -0,0 +1,705 @@
+{
```

Comment:
<p>maybe the phrasing is a bit weird. "We would need to add data" sound more intuitive.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/argilla/pull/5183/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>